doc/api/develop.md

```diff
@@ -0,0 +1,31 @@
+# Unreleased API
+
+```{caution} The following APIs are under development.
+    They are subject to change without notice.
+```
```

Comment:
Aren't these for compatibility with the old structure? I was viewing `develop.md` more as a developer's api documentation for extending spglib itself.

Reply:
Maybe no need to warn in `develop.md`. This is just a reminder for someone getting lost in this place.

Reply:
I think `develop.md` is for the developers. Then, are there more appropriate callouts?

Reply:
```suggestion
# Developer's API

These APIs are for the internal usage and development of Spglib itself.
```